### ISSUE TITLE
Current RCS Implementation is horribly bad. Standalone RCS needs to be implemented.

### ISSUE THREAD

**Issue by Spamm00r** [TKazer/CS2_External#143]
The current RCS implmentation is horribly bad. When aiming at head, only the first shot hits, all others shoot over the head and miss.

This is because RCS only activates when crosshair is on the target. But that is only the case for the first shot. After the first shot, recoil moves the crosshair above head and crosshair is not on target anymore and RCS does not work missing all further shots.

Solution: You need to implement standalone RCS. So that even shooting at wall,  recoil is compensated and all shots land at the same point of aim. Thatway, also shots at enemy will always land at the desired point.

**Comment by KGAT1W** [comment]
This is external cheat. 
There is best rcs for ext hack!

**Comment by Spamm00r** [comment]
No. Thats not true. You can have much better RCS in external hack to.

All you have to do is make RCS standalone and not depend on having an aiming target.

**Comment by mildnsj** [comment]
> No. Thats not true. You can have much better RCS in external hack to.
> 
> All you have to do is make RCS standalone and not depend on having an aiming target.

Then do it yourself, this is only a opensource learning project
